rust/gui-client/src-tauri/src/client.rs

```diff
@@ -123,14 +131,28 @@ fn fix_log_filter(settings: &mut AdvancedSettings) -> Result<()> {
 /// Don't drop the log handle or logging will stop.
 fn start_logging(directives: &str) -> Result<common::logging::Handles> {
     let logging_handles = common::logging::setup(directives)?;
+    let git_version = firezone_bin_shared::git_version!("gui-client-*");
+    let system_uptime_seconds = firezone_headless_client::uptime::get().map(|dur| dur.as_secs());
     tracing::info!(
         arch = std::env::consts::ARCH,
         os = std::env::consts::OS,
         ?directives,
-        git_version = firezone_bin_shared::git_version!("gui-client-*"),
-        system_uptime_seconds = firezone_headless_client::uptime::get().map(|dur| dur.as_secs()),
+        ?git_version,
+        ?system_uptime_seconds,
         "`gui-client` started logging"
     );
+    sentry::add_breadcrumb(Breadcrumb {
+        ty: "logging_start".into(),
+        category: None,
+        data: {
+            let mut map = BTreeMap::default();
+            map.insert("directives".into(), directives.into());
+            map.insert("git_version".into(), git_version.into());
+            map.insert("system_uptime_seconds".into(), system_uptime_seconds.into());
+            map
+        },
```

Comment:
Maps can be initialised from tuple-arrays in a one-liner:

```suggestion
        data: BTreeMap::from([
	    ("directives".into(), directives.into()),
	    ("git_version".into(), git_version.into()),
            ("system_uptime_seconds".into(), system_uptime_seconds.into())
	  ]),
```